Formula/isync.rb

```diff
@@ -37,6 +37,45 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    If you setup your ~/.mbsync file with a Group called "Periodic", 'brew services start isync' will sync that group every five minutes.  Typically, those rules would check your inbox, trash, and sent mail.
```

Comment:
You can add another commit to this branch, or amend the first commit and force-push.